Gruntfile.js

```diff
@@ -3,6 +3,8 @@
  */
 module.exports = function(grunt) {
 
+  const sass = require('node-sass');
```

Comment:
This is required for newer versions of grunt-sass so it will be here when we are ready for it. For now, I hard wired the version to the previous version as there were compilation errors due to changes in syntax.